openedx/features/enterprise_support/enrollments/utils.py

```diff
@@ -62,7 +61,6 @@ def lms_enroll_user_in_course(
                 enrollment_attributes=None,
                 enterprise_uuid=enterprise_uuid,
             )
-            add_user_to_course_cohort(cohort_name, course_id, user)
```

Comment:
any use of the .cohorts package here causes a circular dep but we can just not even call this since effecitvely we don't even use it in bulk enrollment right now